### PR TITLE
Bugfix/add sympy 1.10 support

### PR DESCRIPTION
**Pull request by cqc-melf**
fixes problem with sympy 1.10

### CODE REVIEW

**Review by cqc-alec — APPROVED** [review]

**Comment by cqc-melf** [comment]
Very simple solution to all the mypy errors, if you want something better please let me know.

File: pytket/mypy.ini [hunk 1]
```diff
@@ -26,3 +26,7 @@ ignore_errors = True
 [mypy-lark.*]
 ignore_missing_imports = True
 ignore_errors = True
+
+[mypy-sympy]
+ignore_missing_imports = True
+ignore_errors = True
```
Comment (cqc-alec) [inline pytket/mypy.ini:32]: Is this still necessary when we have the "type:  ignore" comments there?

Comment (cqc-melf) [reply pytket/mypy.ini]: Thank you, it is not. Removed it.

**Review by cqc-alec — APPROVED** [review]
Thanks, I will make a new issue to try and fix up the sympy type annotations, but let's go with this for now.

**Pull request merged**